src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java

```diff
@@ -939,8 +939,8 @@ public void testLoggedTime() throws IOException {
         final List<MessageLevelPair> loggedMessages = antTask.getLoggedMessages();
 
         assertLoggedTime(loggedMessages, testingTime, "Total execution");
-        assertLoggedTime(loggedMessages, testingTime, "To locate the files");
-        assertLoggedTime(loggedMessages, testingTime, "To process the files");
+        assertLoggedTime(loggedMessages, testingTime, "File discovery took");
+        assertLoggedTime(loggedMessages, testingTime, "File processing took");
```

Comment:
you can not `take away` without `breaking`
you can not spell `clean` without `lean`